newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -557,54 +575,140 @@ export default class ThemableEventsTree extends Component<EventsTreeProps, *> {
     );
   };
 
+  _temporallyUnfoldNode = (isOverLazy: boolean, node: SortableTreeNode) => {
+    if (!node.event) return;
+
+    const isNodeTemporallyUnfolded = this.temporallyUnfoldedNodes.some(
+      foldedNode => node.key === foldedNode.key
+    );
+    if (isOverLazy) {
+      if (!this._hoverTimerId && !node.expanded) {
+        if (node.event && !isNodeTemporallyUnfolded) {
```

Comment:
the check on event presence shoud not be necessary here since you're doing it first thing in the method.

Can destructuring with 
```js
const {event} = node;
if (!event) return;
```
help you?